modules/dicom_archive/php/dicomarchiveanonymizer.class.inc

```diff
@@ -0,0 +1,62 @@
+<?php
+/**
+ * This class implements a data mapper which anonymizes Patient Name
+ * or patient ID in the DICOM Archive based on the LORIS configuration.
+ *
+ * PHP Version 7
+ *
+ * @category   Behavioural
+ * @package    Main
+ * @subpackage Imaging
+ * @author     Dave MacFarlane <david.macfarlane2@mcgill.ca>
+ * @license    http://www.gnu.org/licenses/gpl-3.0.txt GPLv3
+ * @link       https://www.github.com/aces/Loris/
+ */
+namespace LORIS\dicom_archive;
+
+/**
+ * A DICOMArchiveAnonymizer is a \LORIS\Data\Mapper which anonymizes improperly
+ * labeled rows in the dicom archive table.
+ *
+ * @category   Behavioural
+ * @package    Main
+ * @subpackage Imaging
+ * @author     Dave MacFarlane <david.macfarlane2@mcgill.ca>
+ * @license    http://www.gnu.org/licenses/gpl-3.0.txt GPLv3
+ * @link       https://www.github.com/aces/Loris/
+ */
+class DICOMArchiveAnonymizer implements \LORIS\Data\Mapper
+{
+    /**
+     * Maps the Patient Name and Patient ID to 'INVALID - HIDDEN' if it doesn't
+     *
```

Comment:
Is there some info missing here? The ending of the phrase doesn't specific what is the exact condition for "if".